lvgl-codegen/Cargo.toml

```diff
@@ -1,12 +1,23 @@
 [package]
 name = "lvgl-codegen"
 version = "0.1.0"
+description = "Code generation based on LVGL source code"
 authors = ["Rafael Caricio <rafael@caricio.com>"]
 edition = "2018"
+license = "MIT"
 publish = false
-
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+build = "build.rs"
 
 [dependencies]
-tera = "1.3.0"
 regex = "1.3.9"
+quote = "1.0.7"
+lazy_static = "1.4.0"
+clang = { path = "../../clang-rs" }
+itertools = "0.9.0"
+proc-macro2 = "1.0.18"
+Inflector = "0.11.4"
+lang-c = "0.8.1"
```

Comment:
Remove this, not used.

Reply:
For context, I tried to use this library. But it cannot parse C99 standard. So I went back to `clang-rs`. If we can find a good C99 parser we could simplify a lot all this code generation work.